packages/url-state-provider/src/index.ts

```diff
@@ -112,8 +128,8 @@ function stateToURL(state) {
  * @param {string} stateID
  * @returns state for stateID
  */
-function currentState(stateID) {
-  return URLToState()[stateID]
+function currentState(stateID: string): object {
+  return URLToState()[stateID] as object
```

Comment:
do you need to cast here as object? maybe you could specify a type here for this type of thing

Reply:
I don't fully understand but thats was linter is saying.